### PR TITLE
Updates README for v2022.2

### PR DESCRIPTION
**Pull request by matsduf**
## Purpose

Preparation for v2022.2 release.

## Changes

Sets correct versions of OS etc.

## How to test this PR

Documentation, to be reviewed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,11 +97,11 @@ at large cloud providers.
...
 ---------------- | --------| ---------------
 CentOS Linux 7   | 5.5     | *not supported*
-Debian 11        | 10.5    | 13.7
+Debian 11        | 10.5 ?? | 13.7 ??
 Docker           | n/a     | n/a
-FreeBSD 13.0     | 5.7 (*) | 13.6
-Ubuntu 22.04     | 10.5    | 14.2
-Rocky Linux 8.6  | 10.3    | 10.19
+FreeBSD 13.1     | 5.7 (*) | 13.9
+Rocky Linux 9.1  | 10.5 ?? | 13.??
```
Comment (blacksponge) [inline README.md:103]: DB version for Rocky Linux: MariaDB 10.5 / PG 13.7

File: README.md [hunk 2]
```diff
@@ -97,11 +97,11 @@ at large cloud providers.
...
+Ubuntu 22.04     | 10.6    | 14.2 ??
```
Comment (blacksponge) [inline README.md:104]: The version of PG for Ubuntu seems to be 14.5 (https://packages.ubuntu.com/jammy/postgresql-14)

File: README.md [hunk 3]
```diff
@@ -97,11 +97,11 @@ at large cloud providers.
 Operating System | MariaDB | PostgreSQL
 ---------------- | --------| ---------------
 CentOS Linux 7   | 5.5     | *not supported*
-Debian 11        | 10.5    | 13.7
+Debian 11        | 10.5 ?? | 13.7 ??
```
Comment (blacksponge) [inline README.md:100]: MariaDB: 10.5
PG: 13.8

Comment (matsduf) [reply README.md:100]: fixed.

File: README.md [hunk 4]
```diff
@@ -117,11 +117,11 @@ Rocky Linux 8.6  | 10.3    | 10.19
...
+Debian 11        | 5.32 ?
 Docker           | (*)
-FreeBSD 13.0     | 5.32
-Ubuntu 22.04     | 5.34
-Rocky Linux 8.6  | 5.26
+FreeBSD 13.1     | 5.32
+Rocky Linux 9.1  | 5.32 ?
+Ubuntu 22.04     | 5.34 ?
 
 * Zonemaster requieres Perl version 5.16 or higher.
```
Comment (blacksponge) [inline README.md:126]: Shouldn't `requieres` be `requires`?

Comment (matsduf) [reply README.md]: Fixed.

**Comment by matsduf** [comment]
@tgreenx, what version Perl version is Debian 11 using?

@mattias-p, changed Rocky Linux version to 8.7. Please update with versions of Perl, MariaDB and PostgreSQL.

**Comment by tgreenx** [comment]
> @tgreenx, what version Perl version is Debian 11 using?

v5.32.1

**Comment by matsduf** [comment]
> > @tgreenx, what version Perl version is Debian 11 using?
> 
> v5.32.1

Thanks. Set as 5.32.

File: README.md [hunk 5]
```diff
@@ -97,11 +99,11 @@ at large cloud providers.
...
 ---------------- | --------| ---------------
 CentOS Linux 7   | 5.5     | *not supported*
-Debian 11        | 10.5    | 13.7
+Debian 11        | 10.5    | 13.8
 Docker           | n/a     | n/a
-FreeBSD 13.0     | 5.7 (*) | 13.6
-Ubuntu 22.04     | 10.5    | 14.2
-Rocky Linux 8.6  | 10.3    | 10.19
+FreeBSD 13.1     | 5.7 (*) | 13.9
+Rocky Linux 8.7  | 10.3 ?? | 10.19 ??
```
[the remark this answers is not in the document]
Comment (matsduf) [reply README.md]: Updated.

File: README.md [hunk 6]
```diff
@@ -119,11 +121,11 @@ Operating System | Perl
 CentOS Linux 7   | 5.16
 Debian 11        | 5.32
 Docker           | (*)
-FreeBSD 13.0     | 5.32
-Ubuntu 22.04     | 5.34
-Rocky Linux 8.6  | 5.26
+FreeBSD 13.1     | 5.32
+Rocky Linux 9.1  | 5.26 ??
```
[the remark this answers is not in the document]
Comment (matsduf) [reply README.md]: Updated.

File: README.md [hunk 7]
```diff
@@ -119,11 +121,11 @@ Operating System | Perl
...
+Ubuntu 22.04     | 5.34 ?
```
[the remark this answers is not in the document]
Comment (matsduf) [reply README.md]: Updated.

File: README.md [hunk 8]
```diff
@@ -119,11 +121,11 @@ Operating System | Perl
 CentOS Linux 7   | 5.16
 Debian 11        | 5.32
 Docker           | (*)
-FreeBSD 13.0     | 5.32
+FreeBSD 13.1     | 5.32
+Rocky Linux 9.1  | 5.26
```
Comment (mattias-p) [inline README.md:125]: `9.1` -> `8.7`

**Review by mattias-p — COMMENTED** [review]
Found a typo but otherwise it looks good.

File: README.md (hunk 8 above)
Comment (matsduf) [reply README.md]: Fixed. Please re-review.

**Review by mattias-p — APPROVED** [review]

**Pull request merged**